client/components/mma/accountoverview/AccountOverview.stories.tsx

```diff
@@ -26,6 +30,9 @@ export const NoSubscription: ComponentStory<typeof AccountOverview> = () => {
 	fetchMock
 		.restore()
 		.get('/api/cancelled/', { body: [] })
+		.get('/mpapi/user/mobile-subscriptions', {
+			body: { subscriptions: [] },
```

Comment:
Do we need these calls to mpapi on the tests which aren't IAPs?

Reply:
yes, since mpapi is still being called

Reply:
> Looks good - just a couple of minor comments. Do we have any tests which handles if the call fails?

Good question, I don't think so but it will just show the "Oops" page as it does if the `me/mma` call fails